WGSmut_pipeline/CGA_WES_CCLE_Characterization_Pipeline_v0.1_Jul2019_copy.wdl

```diff
@@ -36,10 +51,10 @@ workflow CGA_Production_Analysis_Workflow {
     File DB_SNP_VCF
     # index file of VCF file of DB SNP variants
     File DB_SNP_VCF_IDX
-    #
+    # catalogue of somatic mutations in VCF format
     File cosmicVCF
     # panel of normals
-    File normalPanel
+    File MuTectNormalPanel
```

Comment:
there are a lot of changes within this file. why is that?

Reply:
it is your own changes from the base cga pipeline. git considered your new file as a renaming from the previous cga wdl that I had

Reply:
I'm confused why/when I had made these changes. I don't remember doing this. I'm not sure but I wonder if I had accidentally pushed some changes that were experimental. I'll see if I can find anything.

Reply:
well don't worry I know where they came from. It is because it was an update from an old CGA pipeline from guillaume

Reply:
but this is our current CGA pipeline, the one we have used for the past 2 releases, I haven't touched anything